data/schemas/validate_playerResponse.json

```diff
@@ -50,25 +50,39 @@
 				{
 					"type": "object",
 					"required": [
-						"accountName",
+						"name",
+						"profileUrl",
 						"pfp",
 						"pfphash",

```

Comment:
Should this be `pfpHash`?

Reply:
Yes it should be. Give me two seconds...